common/debugging.ts

```diff
@@ -16,5 +16,5 @@ export const DEBUGGING_OPTIONS = {
   // (instead of waiting for 10-30 seconds)
   SHOW_HINT_AFTER_ONE_SECOND: true,
 
-  IGNORE_ASYNC_STORAGE: false,
+  IGNORE_ASYNC_STORAGE: true,
```

Comment:
should return the debugging flags you used to its original value as it can cause conflicts further on.

Reply:
will do